pkg/models/server_group.go

```diff
@@ -321,7 +321,7 @@ func (self *ServerGroup) AddServer(coordConn zkhelper.Conn, s *Server) error {
 		}
 	} else if s.Type == SERVER_TYPE_SLAVE && len(masterAddr) > 0 {
 		// send command slaveof to slave
-		err := utils.SlaveOf(s.Addr, masterAddr)
+		err := utils.SlaveOf(s.Addr, masterAddr, password, masterPassword)
```

Comment:
qdb and reborn-server will not use proxy password, they only use server password